src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -30,7 +30,7 @@
     <Compile Include="OpenTests.cs" />
     <Compile Include="ImportExportTests.cs" />
     <Compile Include="PropertyTests.cs" />
-    <Compile Include="RSACngTests.cs" />
```

Comment:
It's also possible that we just never noticed, since we don't run build.sh in CI yet.